### ISSUE TITLE
quastation about rpl

### ISSUE THREAD

**Issue by yamini4real** [iot-lab/iot-lab#269]
hi
i have a quastation
is it possible to change the routing metric to something different than ETX ?
if so can u help me how can i do such work?

**Comment by aabadie** [comment]
Can you give more details ? What OS are you using ?

**Comment by yamini4real** [comment]
yes sir i want to do it with rpl some m3 node and some a8 nodes using contiki os

**Comment by yamini4real** [comment]
why no one answears?
